### ISSUE TITLE
0.14 doesn't build on windows

### ISSUE THREAD

**Issue by ordian** [rust-rocksdb/rust-rocksdb#411]
Tried with msvc 2015, 2017 and 2019 toolchains, same error
```
running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.25.28610\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MT" "-O2" "-I" "rocksdb/include/" "-I" "rocksdb/" "-I" "rocksdb/third-party/gtest-1.8.1/fused-src/" "-I" "snappy/" "-I" "." "-W4" "-EHsc" "-DSNAPPY=1" "-DNDEBUG=1" "-DHAVE_PCLMUL=1" "-DHAVE_SSE42=1" "-DOS_WIN=1" "-DROCKSDB_WINDOWS_UTF8_FILENAMES=1" "-FoD:\\a\\openethereum\\openethereum\\target\\release\\build\\librocksdb-sys-ddc6bb4f3d83ba77\\out\\rocksdb/env/fs_posix.o" "-c" "rocksdb/env/fs_posix.cc"
persistent_cache_tier.cc
fs_posix.cc
rocksdb/env/fs_posix.cc(9): fatal error C1083: Cannot open include file: 'dirent.h': No such file or directory

error occurred: Command "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.25.28610\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MT" "-O2" "-I" "rocksdb/include/" "-I" "rocksdb/" "-I" "rocksdb/third-party/gtest-1.8.1/fused-src/" "-I" "snappy/" "-I" "." "-W4" "-EHsc" "-DSNAPPY=1" "-DNDEBUG=1" "-DHAVE_PCLMUL=1" "-DHAVE_SSE42=1" "-DOS_WIN=1" "-DROCKSDB_WINDOWS_UTF8_FILENAMES=1" "-FoD:\\a\\openethereum\\openethereum\\target\\release\\build\\librocksdb-sys-ddc6bb4f3d83ba77\\out\\rocksdb/env/fs_posix.o" "-c" "rocksdb/env/fs_posix.cc" with args "cl.exe" did not execute successfully (status code exit code: 2).
```

**Comment by ordian** [comment]
To be precise, the problem is with `librocksdb-sys`, note that it can be automatically updated from 6.2 to 6.7 in `Cargo.lock` even for rocksdb 0.13, because these versions are semver compatible. But RocksDB doesn't follow semver, so using the same versions for the `-sys` crate causes problems. They could be alleviated by pinning a certain version of the `-sys` crate [here](https://github.com/rust-rocksdb/rust-rocksdb/blob/eac44e64da966bc0070a667db92cea55851fd342/Cargo.toml#L29).

**Comment by aleksuss** [comment]
Yes. I suppose I know how to fix it. I create a PR soon.

**Comment by aleksuss** [comment]
@ordian i've released `librocksdb-sys 6.7.4` with fix. It should help if you use `rocksdb` as dependency.

**Comment by ordian** [comment]
Fixed by #412 and released in #413.

**Comment by ordian** [comment]
@aleksuss thanks for the quick fix!
